client/modules/IDE/components/About.jsx

```diff
@@ -28,7 +28,7 @@ function About(props) {
         />
         <div className="about__content-column">
           <p className="about__version-info">
-            Web Editor: <span>v{packageData?.version}</span>
+            {t('About.WebEditor')}: <span>v{packageData?.version}</span>
```

Comment:
I have a question about this.  Does Hindi and other languages use the same "v1.0" syntax?  The "v" stands for "version" so I don't know if that is universal or if it might be different in some languages.

Reply:
I think it depends on how you want to present it to the users. Since it is written as `Web Editor: v{version-number}` in english and not like `Web Editor: version {version-number}`. So it should be fine in other languages also if this convention of specifying the versions is followed.

Reply:
> I have a question about this. Does Hindi and other languages use the same "v1.0" syntax? The "v" stands for "version" so I don't know if that is universal or if it might be different in some languages.

yes! hindi speakers can understand 'v' is used for 'version'